### DISCUSSION TITLE
[Question] How to show controll button each widget. I can't see like video preview

### DISCUSSION THREAD

**Discussion by Hieuntdz** [hm21/pro_image_editor#276] [Q&A]

**Comment by Hieuntdz** [comment]
<img width="467" alt="Screenshot 2024-12-01 at 21 23 49" src="https://github.com/user-attachments/assets/1cf39c8b-50de-474d-84fd-60e87ee61a18">

as this screenshot. it's icon close, roate, edit. I can't see or show theme

**Comment by hm21** [comment]
By default, these buttons are only visible on desktop devices and remain hidden on mobile devices. If you want to make them always visible regardless of the platform, you can achieve this as shown below:

```dart
configs: ProImageEditorConfigs(
    layerInteraction: const LayerInteraction(
      selectable: LayerInteractionSelectable.enabled,
    ),
),
```
